ui/components/ui/add-network/add-network.component.js

```diff
@@ -0,0 +1,96 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import ActionableMessage from '../actionable-message';
+
+const AddNetwork = ({
+  onBackClick,
+  onAddNetworkClick,
+  onAddNetworkManuallyClick,
+  featuredRPCS,
+}) => {
+  const t = useContext(I18nContext);
+
+  const onBack = () => {
+    onBackClick();
+  };
+
+  const onAddNetwork = () => {
+    onAddNetworkClick();
+  };
+
+  const onAddNetworkManually = () => {
+    onAddNetworkManuallyClick();
+  };
+
+  const nets = featuredRPCS
+    .sort((a, b) => (a.ticker > b.ticker ? 1 : -1))
+    .slice(0, 5);
+
+  return (
+    <div className="add-network__wrapper">
```

Comment:
[Q] Could any of these divs with custom styles be replaced with the `<Box>` component to cut back on new styles we have to maintain?

Reply:
Yes, it could. 
Fixed in: bf446bf677fefe3b46b4fb3b60630f531939dd36